assets/scripts/main.js

```diff
@@ -86,7 +86,7 @@
   $(document).ready(() => {
     // Disable copy/paste actions on billing email fields.
     if (shop_standards_settings.emailConfirmationEmail === 'yes') {
-      $('#billing_email, #billing_email_confirmation').each(function() {
+      $('#billing_email, #billing_email_confirmation').each(function x() {
```

Comment:
Why did you name this function?

Reply:
The linter was requiring it, I'll double check.

Reply:
Confirmed: it was needed by the linter as a "warning". I've now disabled the "func-names" check in `.eslintrc` so we don't get the warning.